ES/ESmy020.xml

```diff
@@ -31,71 +31,394 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-020</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Maṣḥafa gǝnzat “Book of the Funeral Ritual” (fols. 2ra l.1–53rb
-                     l.10)&#xD; - Homily of St Jacob of Serug on funeral of priests and deacons
-                     (fols. 11rb l.3–15rb l.6)&#xD; - Homily of ʾAbbā Salāmā, Metropolitan of
-                     Ethiopia (fols. 32rb l.10–35rb l.10)&#xD; II) Maṣḥafa krǝstǝnnā “Book of the
-                     Christening”, called here Maṣḥafa ṭǝmqat (fols. 54ra l.1–71rb l.8). For the
-                     title of the book s. e.g., EMML 3107 (EMML VIII, 94)&#xD; &#xD; Additiones
-                     1-2.</summary>
+                  <summary>
+                  </summary>
+
+
                   <msItem xml:id="ms_i1">
+                    <locus from="1r" to="74rb"/>
+                    <title type="complete" ref="LIT1931Mashaf"/>
 
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                    <msItem xml:id="ms_il.1">
+                    <locus from="2ra" to="2vb"/>
+                    <title type="incomplete">Mǝlṭān- and ʾaryām-antiphons for priests and deacons</title>
+                    <incipit>
+                    <q xml:lang="gez">
+                      </q></incipit>
+                      <explicit>
+                      <q xml:lang="gez">ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ
+                      </q></explicit>
+                    <note>The text is the end of <ref corresp="#ms_il.6"/>; the leaf is placed in the reversed position,
```

Comment:
you may
(a) leave it as it is, it is OK and has been used; even better would be to use relations to specify the link

**NOTE that it must be `<ref target>` not `<ref corresp>`!!! With this mistake the file is not valid**


(b) reconstruct the msItems as you did in many other cases, that is have at one place
```
<msItem xml:id="ms_il.6">
                          <locus from="5ra" to="6vb"/><locus from="2ra" to="2vb"/>
                          <title type="complete">Mǝlṭān and ʾaryām-antiphons for the priests</title>
                          <incipit xml:lang="gez"><locus target="#5ra"/><hi rend="rubric">እሰግድ፡ ለከ፡ አብ፡ ቀዳማ</hi>ዊ፡ እሰግድ፡ ለከ፡ ወልድ፡ ዋህድ፡
                              ንሰግድለከ፡ ኦመንፈስ፡ ቅዱስ፨ ወሐሥሥ፡ በኃቤከ፡ በእንተ፡ ነፍስ፡ ገብርከ፡ እገለ፡ ዘወጽአ፡ እምዝንቱ፡ ሥጋ፡ ከመ፡ ኵሎሙ፡ አበዊነ፡
                              ለይኩን፡ ሎቱ፡ ዕረፍተ፡</incipit>
                          
                          <quote xml:lang="gez"><locus target="#6vb"/>ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ<note> the text continues on <locus target="#2ra"/> </note>ን፡ ኅቡረ፡ ይባርክዎ፡ ለአብ፨</quote>
                          <explicit xml:lang="gez"><locus target="#2vb"/>ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ</explicit>
                      </msItem>
```